src/status_im2/contexts/shell/view.cljs

```diff
@@ -38,38 +41,77 @@
+                      :margin-left   20}}
+   (i18n/label :t/jump-to)])
+
+(defn render-card [{:keys [id type content] :as card}]
+  (let [card-data (case type
+                    constants/one-to-one-chat-card
+                    @(re-frame/subscribe [:shell/one-to-one-chat-card id])
+
+                    constants/private-group-chat-card
+                    @(re-frame/subscribe [:shell/private-group-chat-card id])
+
+                    constants/community-card
+                    (if content
+                      @(re-frame/subscribe [:shell/community-channel-card
+                                            id (get-in content [:data :channel-id])
+                                            content])
+                      @(re-frame/subscribe [:shell/community-card id])))]
+    [switcher-cards/card (merge card card-data)]))
+
+(defn jump-to-list [switcher-cards shell-margin]
+  (if (seq switcher-cards)
+    [rn/flat-list
+     {:data                 switcher-cards
+      :render-fn            render-card
+      :key-fn               :id
+      :header               (jump-to-text)
+      :num-columns          2
+      :column-wrapper-style {:margin-horizontal shell-margin
+                             :justify-content   :space-between
+                             :margin-bottom     16}
+      :style                {:top      0
+                             :left     0
+                             :right    0
+                             :bottom   -1
+                             :position :absolute}}]
+    [placeholder]))
+
 (defn shell []
-  [safe-area/consumer
-   (fn [insets]
-     [rn/view {:style {:top              0
-                       :left             0
-                       :right            0
-                       :bottom           -1
-                       :position         :absolute
-                       :background-color colors/neutral-100}}
-      [common.home/top-nav {:type  :shell
-                            :style {:margin-top (:top insets)}}]
-      [placeholder]
-      [rn/scroll-view {:style {:padding-horizontal 20
-                               :flex-direction     :row}}
-       [quo/text {:size   :heading-1
-                  :weight :semi-bold
-                  :style  {:color      colors/white
-                           :margin-top 12}}
-        (i18n/label :t/jump-to)]]])])
+  (let [switcher-cards @(re-frame/subscribe [:shell/sorted-switcher-cards])
+        width          @(re-frame/subscribe [:dimensions/window-width])
+        shell-margin   (/ (- width 320) 3)] ;; 320 - two cards width
+    [safe-area/consumer
+     (fn [insets]
+       [rn/view {:style {:top              0
+                         :left             0
+                         :right            0
+                         :bottom           -1
+                         :position         :absolute
+                         :background-color colors/neutral-100}}
+        [common.home/top-nav {:type  :shell
```

Comment:
quo/top-nav

Reply:
we don't have top-nav as quo component anymore (https://github.com/status-im/status-mobile/issues/14408)